components/bl-sweet-alert/src/index.js

```diff
@@ -0,0 +1,73 @@
+import { useCallback, useEffect, useMemo, useRef } from 'react';
+import Swal from './lib/sweetalert2';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function SweetAlertComponent({ component, eventHandlers, elRef }) {
+  const {
+    style, classList, display, position, title, text, icon, iconColor, footer, grow, backdrop,
+    allowOutsideClick, allowEscapeKey, allowEnterKey, buttonsSettings, additionalSettings,
+  } = component;
+
+  const { onConfirm, onDeny, onDismiss } = eventHandlers;
+
+  const swalInstanceRef = useRef(null);
+
+  const options = useMemo(() => ({
+    position, title, text, icon, iconColor, footer, grow, backdrop, allowOutsideClick,
+    allowEscapeKey, allowEnterKey, ...buttonsSettings, ...additionalSettings,
+  }), [position, title, text, icon, iconColor, footer, grow, backdrop, allowOutsideClick,
+    allowEscapeKey, allowEnterKey, buttonsSettings, additionalSettings]);
+
+  const showAlert = useCallback(() => {
+    if (elRef.current && !swalInstanceRef.current) {
+      swalInstanceRef.current = createAlert({ target: elRef.current, ...options });
+      swalInstanceRef.current.then((response) => {
```

Comment:
In fact, value will not always be true, for example, the value of the input can be stored there, in addition, in certain cases this parameter will not exist at all, so this information is useful and will help the customer take into account the received response when interacting with the alert.

Reply:
> In fact, value will not always be true

If I understand correctly, Serhiy is about next lines:

`isConfirmed && onConfirm({ response });`

What the reason to pass `isConfirmed` to `onConfirm` event if we already check it for positive. It could be string/array/object?